module.exports = function handleErrs(err) {
    console.log("Caught Err");
    console.log(err);
};
